modules/bene_core/bene_core.install

```diff
@@ -53,3 +53,12 @@ function bene_core_update_8501() {
   $config_storage = \Drupal::service('config.storage');
   $config_storage->write('embed.button.media_browser', $source->read('embed.button.media_browser'));
 }
+
+/**
+ * Give Bene Features configuration access to staff admin users.
+ */
+function bene_core_update_8502() {
+  $staff_admin = Role::load('staff_admin');
```

Comment:
Can this role ever be deleted?

Reply:
Theoretically, yes. Seems unlikely we'd run into trouble though, since this will go out and get run on the existing sites. Still, probably best practice to test if the results are empty ;)